src/Concerns/HandlesGraphqlRequests.php

```diff
@@ -121,13 +121,11 @@ public function fieldFromResolver($source, $args, $context, ResolveInfo $info)
         $className = $this->resolveClassName($info);
         $methodName = $this->resolveMethodName($info);
 
-        try {
-            $resolver = app($className);
-        } catch (ReflectionException $e) {
-            // NOTE: It's OK if the class to reflect does not exist
-            $resolver = null;
+        if (! app()->has($className) && ! class_exists($className)) {
```

Comment:
Nitpicking here but I do find "if not x nor y" a bit easier to read than "if not x and not y".

```suggestion
        if (! (app()->has($className) || class_exists($className))) {
```

On the other hand, maybe the early return here isn't necessary and we could just wrap it in the statement.

```
if (app()->has($className) || class_exists($className)) {
    $resolver = app($className);
    if (method_exists($resolver, $methodName)) {
        return $resolver->{$methodName}($source, $args, $context, $info);
    }
}
```